### PR TITLE
Init project, add Allowance Module deployments

### PR DESCRIPTION
**Pull request by mikhailxyz**
Similar structure to https://github.com/gnosis/safe-deployments

Allowance module deployments were taken from https://github.com/gnosis/safe-modules/blob/master/allowances/networks.json

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+const allowanceModule = getAllowanceModuleDeployment()
+
+// Returns latest contract version, even if not finally released yet
+const allowanceModuleNightly = getAllowanceModuleDeployment({ released: undefined })
+
+// Returns released contract version for specific network
+const allowanceModuleGörli = getAllowanceModuleDeployment({ network: "5" })
+
+// Returns released contract version for specific version
+const allowanceModule100 = getAllowanceModuleDeployment({ version: "0.1.0" })
```
Comment (rmeissner) [inline README.md:54]: shouldn't this be `allowanceModule010`? :D

File: src/assets/allowance-module/v0.1.0/allowance-module.json [hunk 2]
```diff
@@ -0,0 +1,717 @@
...
+  "contractName": "AllowanceModule",
+  "version": "0.1.0",
+  "networkAddresses": {
+    "1": "0xCFbFaC74C26F8647cBDb8c5caf80BB5b32E43134",
+    "4": "0xCFbFaC74C26F8647cBDb8c5caf80BB5b32E43134",
+    "5": "0xCFbFaC74C26F8647cBDb8c5caf80BB5b32E43134",
+    "42": "0xCFbFaC74C26F8647cBDb8c5caf80BB5b32E43134",
+    "56": "0xCFbFaC74C26F8647cBDb8c5caf80BB5b32E43134",
+    "100": "0xCFbFaC74C26F8647cBDb8c5caf80BB5b32E43134",
+    "137": "0x1Fb403834C911eB98d56E74F5182b0d64C3b3b4D",
```
Comment (rmeissner) [inline src/assets/allowance-module/v0.1.0/allowance-module.json:12]: Why did we not use the normal deterministic deployment for that one?

**Review by rmeissner — APPROVED** [review]
Looks good in general, the question for the polygon address would probably been better on the safe-modules repo.

File: src/assets/allowance-module/v0.1.0/allowance-module.json (hunk 2 above)
Comment (mikhailxyz) [reply src/assets/allowance-module/v0.1.0/allowance-module.json:12]: @dasanra told me that polygon RPCs had enforced eip155 when he tried to deploy it so I decided to use safe-singleton-factory without testing the non-eip-155 one. It shouldn't affect anything

Comment (dasanra) [reply src/assets/allowance-module/v0.1.0/allowance-module.json:12]: Yes I found this blocker when trying to deploy it on Polygon before. It was enforcing me to add a chain Id, at least in December/January that I tried

**Pull request merged**